backend/src/test/java/ai/verta/modeldb/ExperimentTest.java

```diff
@@ -1816,7 +1816,7 @@ public void deleteExperiments() {
       } catch (StatusRuntimeException e) {
         Status status = Status.fromThrowable(e);
         LOGGER.info("Error Code : " + status.getCode() + " Error : " + status.getDescription());
-        assertEquals(Status.PERMISSION_DENIED.getCode(), status.getCode());
+        assertEquals(Status.NOT_FOUND.getCode(), status.getCode());
```

Comment:
If Invalid experimentId set in request which is not exists in database so return NOT_FOUND